### DISCUSSION TITLE
Status 7590 mit 7.2x ?

### DISCUSSION THREAD

**Discussion by Boxbox99** [Freetz-NG/freetz-ng#227] [Allgemein] [answered]
Hallo zusammen,
ich habe schon gesucht habe aber leider nichts gefunden.
Ich möchte von der Version 7.12 auf die 7.25  (FritzBox 7590) umsteigen. Ich weiß jetzt aber nicht ob das Problem mit dem watchdog oder untrusted noch besteht. Ich habe nur eine Stelle gefunden wo das in dem Config noch auftaucht unter REMOVE_LEFTOVER.
Was muss aktuell eine funktionierende Version eingestellt werden ?

Vielen dank im Vorraus
Michael

**Comment by kainers1** [comment, marked as answer]
Ja funktioniert, alle Details siehe https://github.com/Freetz-NG/freetz-ng/issues/120#issuecomment-787992787
Einstellen musst du nichts besonderes.

**Reply by Gismotro** [reply]
Es gibt aber wohl neue Probleme ab der 7.24 : #224  oder [IPPF](https://www.ip-phone-forum.de/threads/root-login-per-telnet-oder-dropbear-nach-freetz-update-auf-fritz-os-7-25-nicht-mehr-m%C3%B6glich.309772/)

aktuell getestet auf einer 7530(Alien 7520) mit 7.25

**Reply by fda77** [reply]
Schön zu lesen dass die 7530 auch funktioniert! Nicht vergessen: Für kernel-module werden noch alte Sourcen verwendet, da schienbar noch niemand bei AVM per mail gebettelt hat ;-)
(Es werden nur Sourcen für 7520 oder 7530 benötigt)

**Comment by Boxbox99** [comment]
Danke, den Artikel hatte ich nicht gefunden.
Wünsche noch ein schönes Wochenende

**Comment by fda77** [comment]
Meine 7590 7.26 lauf auch sei ~2 Wochen gut. Abgesehen von dem Mist den AVM verbockt hat, zb dass am Hybridanschluss kein Telekom-Voip mehr funktioniert...
